pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation.go

```diff
@@ -603,6 +603,11 @@ func podSumRessourceRequirements(pod *corev1.Pod) corev1.ResourceRequirements {
 	return ressourceRequirement
 }
 
+type injectionResourceRequirementsDecision struct {
+	skipInjection bool
+	message       string
```

Comment:
small comment
this could have been just a `error` 
`if err == nil` it means the injection is possible. and if `error != nil` we use the error message as value for the annotation

Reply:
Fare remarks.
But now with golang error wrapping we can easily differentiate to kind of error, so we can imaging to make the difference between functional and unexpected errors.

I think we can keep it like this for now.